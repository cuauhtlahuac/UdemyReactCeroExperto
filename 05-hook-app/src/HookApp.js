import React from 'react';
import { CounterApp } from 'component/01-useState/CounterApp';
import { CounterWithCustomHooks } from 'component/CounterWithCustomHooks';

export const HookApp = () => {
	return (
		<div className="container">
			<nav className="navbar navbar-expand-lg navbar-light
					   	    bg-light row my-md-5 my-sm-3">
				<h1>Hook App</h1>
			</nav>
			<section className="mb-5">
				<CounterApp />
			</section>
			<section>
				<CounterWithCustomHooks />
			</section>
		</div>
	);
};
