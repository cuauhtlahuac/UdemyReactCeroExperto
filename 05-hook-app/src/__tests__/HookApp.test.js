import React from 'react';
import { shallow } from 'enzyme';

import HookApp from 'HookApp';

describe('HookApp Tests', () => {
  it('should render correctly', () => {
    const component = shallow(<HookApp />)

    expect(component).toMatchSnapshot();
  })
})
