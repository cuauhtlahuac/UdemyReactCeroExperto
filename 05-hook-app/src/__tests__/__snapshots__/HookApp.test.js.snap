// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp Tests should render correctly 1`] = `
<div
  className="container"
>
  <nav
    className="navbar navbar-expand-lg navbar-light bg-light row my-md-5 my-sm-3 ali justify-content-center"
  >
    <h1>
      Hook App
    </h1>
  </nav>
  <section
    className="mb-5"
  >
    <TodoApp />
  </section>
</div>
`;
